README.md

```diff
@@ -2,46 +2,71 @@
 
 # avere-europarl
 
-scrape https://www.europarl.europa.eu/meps/
+Scrape https://www.europarl.europa.eu/meps/
 
-## TODO - scraper
+## Usage
```

Comment:
First in this section should be the `./start.sh` script that runs docker. That's the simplest way to run this, so that should be the first option.